src/js/components/ComboBox/Combobox.tsx

```diff
@@ -135,13 +145,9 @@ const ComboBox = ({
     }
   }, [selected]);
 
-  // Run this when the flyout visibility
   React.useEffect(() => {
-    if (!flyoutVisible) {
```

Comment:
We should still keep this so that when clicking outside of the combobox will filter the list.